src/components/button/Button.stories.tsx

```diff
@@ -60,5 +63,21 @@ storiesOf("Button", module)
         <div>
             <Button key="link" link onClick={action("delete click")} icon={{shape: "trash"}}/>
             <Button key="link" link onClick={action("copy click")} icon={{shape: "copy"}}/>
+            <Button key="link" link onClick={action("window-close click")} icon={{shape: "window-close"}}/>
+            <Button key="link" link onClick={action("refresh click")} className="icon-button-border" icon={{shape: "refresh"}}/>
+            <Button key="link" link onClick={action("settings click")} className="icon-button-background" icon={{shape: "settings"}}/>
+        </div>
+    ))
+    .add("Buttons Disabled", () => (
+        <div>
+            <Button key="disabled" disabled >
+                {"Disabled"}
+            </Button>
+            <Button key="disabled" primary disabled >
+                {"Primary"}
+            </Button>
+            <Button key="disabled" className="warning-button" disabled >
```

Comment:
Please use `state` prop for Button if we want to render warning button
export enum ButtonState {
    INFO = "info",
    SUCCESS = "success",
    WARNING = "warning",
    DANGER = "danger",
}